src/test/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotificationTest.java

```diff
@@ -81,10 +84,35 @@ void shouldCallSendEmailForSupportingDocuments() {
         verify(commonContent).templateVarsForApplicant(data, data.getApplicant1(), data.getApplicant2());
     }
 
+    @Test
+    @SuppressWarnings("squid:S6068")
+    void shouldCallSendEmailToApplicant2ForSupportingDocuments() {
+        CaseData data = validApplicant2CaseData();
+        data.getApplication().getMarriageDetails().setMarriedInUk(YesOrNo.NO);
+        data.getApplication().setApplicant2CannotUploadSupportingDocument(Set.of(NAME_CHANGE_EVIDENCE));
+
+        final HashMap<String, String> templateVars = new HashMap<>();
+        when(commonContent.templateVarsForApplicant(data, data.getApplicant2(), data.getApplicant1())).thenReturn(templateVars);
```

Comment:
do you want to return emptyMap() here ?

Reply:
@nitinprabhu yeah this is the same method most of the other notification unit tests have used to mock templateVarsForApplicant

Reply:
what I meant was you can replace with below
```when(commonContent.templateVarsForApplicant(data, data.getApplicant1(), data.getApplicant2())).thenReturn(Collections.emptyMap());```